### ISSUE TITLE
Add clock in/out feature just like what we can do in emacs

### ISSUE THREAD

**Issue by hekinami** [orgzly/orgzly-android#1050]
Would you like to consider adding this feature in future?

And if feature like org-pomodoro could be added, it will be very useful.

**Comment by u3frajaeian** [comment]
I can fix this issue

**Comment by hekinami** [comment]
> I can fix this issue

Very glad to hear you are interested in this feature, looking forward to your good news

**Comment by u3frajaeian** [comment]
Hi, can you give me some details?

**Comment by hekinami** [comment]
> Hi, can you give me some details?

I my opinion, we may add one or two extra icon(s) is the menu displayed when we slide right on an item. (from which menu we can change the item state from TODO to DONE)

1. If we click the icon, we can clock in/clock out. 
2. We need add LOGBOOK drawer into the corresponding and the clockin/out history should be logged. 
3. And we need show the result somewhere. I think adding a new section under the meta data block for LOGBOOK drawer might be a good idea. And because log entries for clockin/out might be multiple, so we may need a label to show the sum of all logged time for one note item (i.e. one org headline).

Pomodoro is one good time management technique, so I think it could be great, if this technique can go along with the clockin/out feature. 

When we start a pomodoro, clockin is triggered, and when a pomodoro ends, clockout is triggered automatically. So we need a timer and when a pomodoro time is out, we should notify the user just like the notification in orgzly.

I think it's better to make the pomodoro feature configurable, for example the length of one pomodoro, and if possible the length of long/short break, could be specified by the user.

Thanks

**Comment by cashpw** [comment]
As far as I'm aware this feature is already merged (#691). However, it hasn't been released yet. It's part of the [v1.8.11 milestone](https://github.com/orgzly/orgzly-android/milestone/37). I've filed #1051 to request a release for v1.8.11.

**Comment by hekinami** [comment]
according to #1051 , does that mean we should move to [orgzly-android-revived](https://github.com/orgzly-revived/orgzly-android-revived) ?

**Comment by cbkerr** [comment]
@hekinami  Yes, you are welcome to try out the community fork and help spread the word!

**Comment by hekinami** [comment]
@u3frajaeian @cashpw I checked v1.8.11 released from orgzly-android-revived, I'm afraid that it's implementation of clock in/out is still very basic and could be streamlined. 

Should we move this issue to orgzly-android-revived?
